### ISSUE TITLE
Use new DataZone.json

### ISSUE THREAD

**Issue by wangzhao0217** [nptscot/nptscot.github.io#212]
Hi Martin,

Could you please update the website using the new Datazone JSON file? It is not in the DataZones folder; instead, it's located in the DataZone_2024-05-23 folder. And the same for School.

**Comment by Robinlovelace** [comment]
@mvl22 nudge on this, should be one-liner to test. Let us know if you need any further info.

**Comment by mvl22** [comment]
> It is not in the DataZones folder; instead, it's located in the DataZone_2024-05-23 folder. And the same for School.

I don't know where that is I'm afraid. Did you mean on the server on a repo or somewhere else. There doesn't seem to be a folder named that in the repos I just looked at.

Ideally please always just give a URL, then things can be copied and pasted, rather than searched about for.

**Comment by Robinlovelace** [comment]
> Ideally please always just give a URL, then things can be copied and pasted, rather than searched about for.

Sure, example URL: https://nptscot.blob.core.windows.net/json/DataZone_2024-05-23/S01006506.json

**Comment by mvl22** [comment]
I've tried that, but the data isn't showing, as it's saying the zone doesn't exist, e.g.:
https://nptscot.blob.core.windows.net/json/DataZone_2024-05-23/S01008479.json
gives a 404.

Is there perhaps also an update to the main layer data, which is currently this?
https://nptscot.blob.core.windows.net/pmtiles/data_zones-2023-12-17.pmtiles

Presumably the two both have to be kept in sync.

**Comment by Robinlovelace** [comment]
That sounds like a data issue. @wangzhao0217 can you check?

**Comment by wangzhao0217** [comment]
Hi Robin, not quite sure how to check, But I suppose the data_zones.pmtiles should be used together with the DataZone_json.zip that was generated at some time?
https://github.com/nptscot/outputdata/releases/download/v2024-06-03/DataZone_json.zip
https://github.com/nptscot/outputdata/releases/download/v2024-06-03/data_zones.pmtiles

**Comment by Robinlovelace** [comment]
Thanks Zhao. @mvl22 can you see if the new .json files work with the new data_zones pmtiles: https://nptscot.blob.core.windows.net/pmtiles/data_zones_2024-05-23.pmtiles

**Comment by mvl22** [comment]
Hmm, changing from `data_zones-2023-12-17.pmtiles` to `data_zones_2024-05-23.pmtiles` gives a 'Wrong magic number for PMTiles archive' console error:

```
evented.ts:151 Error: Wrong magic number for PMTiles archive
    at index-2.11.0.js:1:16311
    at Generator.next (<anonymous>)
    at s (index-2.11.0.js:1:420)
```

and the layer is not clickable.

I think we had this before. I'll see if I can dig around and find what that means.

**Comment by Robinlovelace** [comment]
Any thoughts on how to debug this @mvl22 ? I can create a new file with a different version of tippecannoe is one thought.

**Comment by mvl22** [comment]
It's definitely an issue with the data.

If you select
https://nptscot.blob.core.windows.net/pmtiles/data_zones_2024-05-23.pmtiles

in the viewer at:
https://pmtiles.io/

you get:

`Wrong magic number for PMTiles archive`.

Do try a different version of Tippecanoe indeed. Please make sure you test it on that site first before posting back here, so that you can verify the data you created is definitely fine.

**Comment by Robinlovelace** [comment]
@wangzhao0217 can you create a new version and test?

**Comment by Robinlovelace** [comment]
Confirmed: 

![image](https://github.com/nptscot/nptscot.github.io/assets/1825120/182c70e0-5c24-45e7-bb39-26d67e4ee65a)

**Comment by Robinlovelace** [comment]
Nothing to debug here until we get new data..

**Comment by wangzhao0217** [comment]
Hi @mvl22 please find the latest data here https://github.com/nptscot/outputdata/releases/download/v2024-06-01/data_zones.pmtiles

**Comment by Robinlovelace** [comment]
Can you try with this @mvl22 https://nptscot.blob.core.windows.net/pmtiles/data_zones_2024-05-23.pmtiles

**Comment by Robinlovelace** [comment]
![image](https://github.com/nptscot/nptscot.github.io/assets/1825120/339d4546-39c7-4ce3-8888-8961a08c22ff)

**Comment by Robinlovelace** [comment]
Before: 

![image](https://github.com/nptscot/nptscot.github.io/assets/1825120/97da045b-994f-40ab-a3f1-4093b7950eee)

**Comment by Robinlovelace** [comment]
Update: waiting on new release with all the changes including new cohesive network @wangzhao0217

**Comment by Robinlovelace** [comment]
Test file, check this exists before uploading: https://nptscot.blob.core.windows.net/json/DataZone_2024-05-23/S01008680.json

**Comment by Robinlovelace** [comment]
S01008680.json

**Comment by Robinlovelace** [comment]
Re-assigning to Zhao as data issue again.

**Comment by wangzhao0217** [comment]
Hey @Robinlovelace @mvl22 the new DataZone can be found here: https://github.com/nptscot/outputdata/releases/download/v2024-07-02/DataZone_json.zip

**Comment by wangzhao0217** [comment]
July data can be found here: https://github.com/nptscot/outputdata/releases/tag/v2024-07-02
but S01008680.json is missing from DataZone data

**Comment by Robinlovelace** [comment]
Update on this: we have diagnosed the cause of the issue. In pair programming session with @wangzhao0217 we identified that the data was being overwritten for each region. The resulting .zip file contained data only for a single region.

**Comment by wangzhao0217** [comment]
@Robinlovelace @mvl22 the lastest data is ready: https://github.com/nptscot/outputdata/releases/download/v2024-07-02/DataZone_json.zip

**Comment by wangzhao0217** [comment]
> S01008680.json

![image](https://github.com/nptscot/nptscot.github.io/assets/74598734/87533f79-3e58-4c62-9409-10856dd5e631)

**Comment by Robinlovelace** [comment]
Thanks Zhao, will try uploading now..

**Comment by Robinlovelace** [comment]
Update from #225 just updated:

![image](https://github.com/nptscot/nptscot.github.io/assets/1825120/b1dd3e42-88c2-4ddc-8e56-3b56041039de)

**Comment by mvl22** [comment]
Discussed on phone call that there is still data work to be done before this becomes ready to update.

Will wait to hear further.

**Comment by Robinlovelace** [comment]
Correct, it's in our court.

**Comment by Robinlovelace** [comment]
Heads-up @mvl22 I'm looking at this now and think the best quick fix at the moment is to remove this functionality. It's not helpful at the moment. We can revisit re-adding but causes a LOT of complexity that is not a priority at this stage.

**Comment by Robinlovelace** [comment]
There are various reasons for this. Number 1 being that currently the way it's written the code assumes that the model works for all of Scotland. But we're building per region, making the 'destination' values way out. We can re-aggregate at the national level but the whole codebase needs an overhaul before we do that, including the SIM on which utility trips are based. We could keep only the commuter trip values but that would be an incomplete feature. Furthermore this feature was never asked for so let's remove for now.

**Comment by mvl22** [comment]
Pull request submitted to disable the popups.

This is implemented simply by commenting-out (removing) the static definition. The web app therefore finds no definition, so doesn't create the popup.

**Comment by Robinlovelace** [comment]
> I guess #225 can be mothballed as well.

Correct. Closed and branch deleted to keep repo tidy-ish.

**Comment by Robinlovelace** [comment]
Closed with https://github.com/nptscot/nptscot.github.io/commit/b13ea92e20a276638fccb9f4f0284220ae0a05d4

**Comment by Robinlovelace** [comment]
Tested. Confirmed: no more popups!
